crates/store/re_remote_store_types/proto/rerun/v0/common.proto

```diff
@@ -0,0 +1,117 @@
+syntax = "proto3";
+
+package rerun.remote_store.v0;
+
+// unique recording identifier. At this point in time it is the same id as the ChunkStore's StoreId
+message RecordingId {
+    string id = 1;
+}
+
+// A recording can have multiple timelines, each is identified by a name, for example `log_tick`, `log_time`, etc.
+message Timeline {
+    string name = 1;
+}
+
+// A time range between start and end time points. Each 64 bit number can represent different time point data
+// depending on the timeline it is associated with.
```

Comment:
This should specify that the range is inclusive etc.

But really, in general, everything in this file that matches an existing rust type should simply copy-paste the doc from the original rust types (they will likely grow into slightly different types over time and so will their docs and that's fine, but that's a good starting point).

Reply:
I somehow missed this comment. Thanks, I've applied 95% of original documentation i.e. whenever it was actually there and made sense to be applied exactly to protobuf docs.